source/04-PR/03.ptx

```diff
@@ -11,14 +11,14 @@
     </objectives>
     <subsection>
     <title>Activities</title>
-    <activity xml:id="activity-3-1">
+    <activity xml:id="Long Division Basic">
```

Comment:
Note that spaces are not allowed in `xml:id`s.  Use a - to separate words.  I will fix all these shortly